pet-store-react-hooks/src/DeletePetModal/DeletePetModal.css

```diff
@@ -0,0 +1,23 @@
+.delete-pet-modal-wrapper {
+  width: 30rem;
```

Comment:
Don't use `rem`'s for spacing use pixels. You can use `rem` for specifying font size.